ember-file-upload/addon/components/file-dropzone.ts

```diff
@@ -31,7 +31,11 @@ interface FileDropzoneArgs {
   cursor?: 'link' | 'none' | 'copy' | 'move';
 
   // actions
-  filter?: (file: UploadFile) => boolean;
+  filter?: (
+    file: UploadFile,
+    files: File[] | DataTransferItem[],
```

Comment:
I don't think we need to expose any DataTransfer items that aren't files here.

Suggestion:
```js
  // file-dropzone.ts
  get files(): File[] {
    const files = this.dataTransferWrapper?.files ?? [];
    if (this.multiple) return files;

    return files.slice(0, 1);
  }
```